official/transformer/transformer_main.py

```diff
@@ -403,54 +447,105 @@ def _check_bleu_files(flags_dict):
         tf.gfile.Exists(flags_dict["bleu_ref"]),
         tf.gfile.Exists(vocab_file_path)])
 
+  flags_core.require_cloud_storage(["data_dir", "model_dir"])
```

Comment:
I must be confused about how this is working, but where does this actually check if TPU=true? In the func definition below, it seems that all directories would be required to be on cloud, regardless of TPU or not?

Reply:
Nope, you're absolutely correct. Thanks for catching this.